modules/test/playwright/tests/frontend-data-set-views-web/pages/view/tabs/VisualizationModesPage.ts

```diff
@@ -152,20 +143,24 @@ export class VisualizationModesPage {
 			.waitFor();
 	}
 
-	async openParentField(path: string[]) {
-		let fullPath = '';
+	async selectField({
+		dataId,
+		fieldName,
+	}: {
+		dataId?: string;
+		fieldName: string;
+	}) {
+		const treeItem = this.fieldSelectModalContainer.locator(
```

Comment:
I think that this only work for first level items. Nested child are not present until clicking into the `>` toggle :thinking: